go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/heroku/docker-registry-client v0.0.0-20190909225348-afc9e1acc3d5
 	github.com/imdario/mergo v0.3.8
 	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
-	github.com/openshift/client-go v0.0.0-20200116152001-92a2713fa240
+	github.com/openshift/client-go v3.9.0+incompatible
```

Comment:
This version name is fishy! Is this right? No adverse effects? @tchughesiv ?

Reply:
there should be no issues with this as the important line in go.mod for this library is this one in the replace section - https://github.com/kiegroup/kie-cloud-operator/blob/ad198e7e68bafeb5e477dbfe1e45df40cbb35520/go.mod#L48

which forces this in go.sum - https://github.com/kiegroup/kie-cloud-operator/blob/ad198e7e68bafeb5e477dbfe1e45df40cbb35520/go.sum#L668

that said, i'm not sure why this line is changing with this PR... maybe carryover from he operator-utils library change?

Reply:
Any thoughts, Marco?

Reply:
I also think the version at the replace section will take the precedence over that version at the require section. 
That line seem to be generated by go mod as we didn't specify the exact version when operator-utils was converted to use go mod from dep. Now it is carried forward.
So, I think we should be ok.

Btw, why do we use a lot of replace sections given that we know exactly what version we want?
If an exact version was listed in operator-utils' require section, could we have less this  type of version concern?

Reply:
Good question, Marco. I'm not sure and maybe Tommy is better placed to answer this question. I suppose we want to be thorough and avoid this project breaking in case operator-utils uses a different version. In other words, avoid tight coupling to that project.